TESTING.md

```diff
@@ -0,0 +1,61 @@
+# Testing 'oneview-sdk' gem source code
+The 'oneview-sdk' source code provides a unit, integration/functional, system testing strategies, and linting tools to ensure its code style.
```

Comment:
provides ~a~ unit, integration/functional **and** system testing strategies. ~, and~ It also provides linting tools (...)